sdks/java/core/src/test/java/org/apache/beam/sdk/io/WriteFilesTest.java

```diff
@@ -235,7 +243,9 @@ public void testExpandShardedWrite() throws IOException {
         Arrays.asList("one", "two", "three", "four", "five", "six"),
         IDENTITY_MAP,
         getBaseOutputFilename(),
-        Optional.of(20));
+        Optional.of(20),
```

Comment:
I'm still not super happy about readability here due to how many parameters there are, and how easy they are to confuse. What do you think about my suggestion to pass WriteFiles directly, so that the call site contains a builder call and it's obvious what parameters are configured with what values?